src/game.h

```diff
@@ -90,6 +90,9 @@ enum GAME_COMMAND {
 	GAME_COMMAND_MODIFY_GROUPS,
 	GAME_COMMAND_KICK_PLAYER,
 	GAME_COMMAND_CHEAT,
+	GAME_COMMAND_PICKUP_GUEST,
+	GAME_COMMAND_PICKUP_STAFF,
+	GAME_COMMAND_BALLOON_PRESS,
```

Comment:
We should add a guard here `GAME_COMMAND_COUNT` and use that for specifying size of tables with game command handlers.